conwaywebapi/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=DESKTOP-5ED6RRO\\SQLEXPRESS;Database=Conway;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
We probably should not commit this string containing details of your machine. We should possibly add another configuration source that is more secure.

Reply:
I think, however, as this code is throwaway and your machine and accounts will be removed at the end of the exercise, we can let this slides for now. I will add an Issue to put in a new configuration source.